charts/node/README.md.gotmpl

```diff
@@ -0,0 +1,208 @@
+<!--
```

Comment:
Add links to http backups from `snapshots.polkadot.io`, please.

Reply:
done